src/tests/activity.test.js

```diff
@@ -1 +1,141 @@
-test.skip('skip', () => {});
+import request from 'supertest';
+import app from '../app';
+import activity from '../models/activity';
+import activitySeed from '../seeds/activity';
+
+const mockActivity = {
+  name: 'Boxing',
+  description: 'vestibulum aliquet ultrices erat tortor sollicitudin mi non mi integer ac neque duis',
+  isActive: false,
+};
+
+const mockActivityExistingName = {
+  name: 'Chandal',
+  description: 'Any description',
+  isActive: false,
+};
+
+const mockLessName = {
+  name: '',
+  description: 'Some text',
+  isActive: true,
+};
+
+const mockLessDescription = {
+  name: 'Name',
+  description: '',
+  isActive: true,
+};
+
+const mockLessIsActive = {
+  name: 'Names',
+  description: 'some description',
+  isActive: '',
+};
+
+beforeAll(async () => {
+  await activity.collection.insertMany(activitySeed);
+});
+
+describe('GET /api/activities', () => {
+  test('The database must return 200 if is not empty of activities', async () => {
+    const response = await request(app).get('/api/activities').send();
+    expect(response.body.data.length).toBeGreaterThan(0);
+    expect(response.status).toBe(200);
+    expect(response.error).toBeFalsy();
+  });
+  test('The status must be 404, if the url is wrong', async () => {
+    const response = await request(app).get('/activities').send();
+    expect(response.status).toBe(404);
+    expect(response.error).toBeTruthy();
+  });
+  test('If are activities, data must said "Complete activities list" ', async () => {
+    const response = await request(app).get('/api/activities').send();
+    expect(response.body.message).toMatch(/Complete activities list/);
+    expect(response.error).toBeFalsy();
+  });
+});
+
+describe('GET BY ID /api/activities/:id', () => {
+  test('The database must return 200 if the id exist', async () => {
+    const response = await request(app).get('/api/activities/64677fdefc13ae39f1753b24').send();
```

Comment:
What happens if we change the admin's ID in the seed file? I think we should use a variable for the ID and we should capture the ID of the first admin of the seed.

Consider this every time we use admin ID